signer/src/storage/postgres.rs

```diff
@@ -691,13 +703,15 @@ impl super::DbRead for PgStore {
             JOIN sbtc_signer.deposit_signers signers USING(txid, output_index)
             WHERE
                 signers.is_accepted
+                AND (transactions.block_height + deposit_requests.lock_time) > $4
```

Comment:
this check and the in memory one seems to differ for the equal case

Reply:
Fixed.